Rename enums in CSSUnitType to CamelCase and remove CSS_ prefix

zakariaridouh opened the pull request:
#### 0c90030cc272e6a9aa802b43c3b17b39dfc0f635
<pre>
Rename enums in CSSUnitType to CamelCase and remove CSS_ prefix
&lt;<a href="https://bugs.webkit.org/show_bug.cgi?id=229622">https://bugs.webkit.org/show_bug.cgi?id=229622</a>&gt;
&lt;<a href="https://rdar.apple.com/82734077">rdar://82734077</a>&gt;

Reviewed by NOBODY (OOPS!).

Expand the names of most enums, and update their uses across the
codebase.
* CSS_S is renamed to Second etc...
* Cm =&gt; Centimeter and In =&gt; Inch etc...

* Source/WebCore/animation/KeyframeEffect.cpp:
(WebCore::isTimelineRangeOffsetValid):
(WebCore::computeMissingKeyframeOffsets):
(WebCore::specifiedOffsetForParsedKeyframe):
* Source/WebCore/animation/WebAnimationTime.cpp:
(WebCore::WebAnimationTime::WebAnimationTime):
* Source/WebCore/css/CSSImageSetOptionValue.cpp:
(WebCore::CSSImageSetOptionValue::create):
* Source/WebCore/css/CSSPrimitiveValue.cpp:
(WebCore::isValidCSSUnitTypeForDoubleConversion):
(WebCore::isStringType):
(WebCore::CSSPrimitiveValue::primitiveType const):
(WebCore::CSSPrimitiveValue::CSSPrimitiveValue):
(WebCore::CSSPrimitiveValue::~CSSPrimitiveValue):
(WebCore::CSSPrimitiveValue::create):
(WebCore::CSSPrimitiveValue::createFontFamily):
(WebCore::CSSPrimitiveValue::createInteger):
(WebCore::CSSPrimitiveValue::conversionToCanonicalUnitsScaleFactor):
(WebCore::CSSPrimitiveValue::doubleValueDividingBy100IfPercentage const):
(WebCore::CSSPrimitiveValue::doubleValueDividingBy100IfPercentageDeprecated const):
(WebCore::CSSPrimitiveValue::doubleValueInternal const):
(WebCore::CSSPrimitiveValue::doubleValueInternalDeprecated const):
(WebCore::CSSPrimitiveValue::stringValue const):
(WebCore::CSSPrimitiveValue::unitTypeString):
(WebCore::CSSPrimitiveValue::serializeInternal const):
(WebCore::CSSPrimitiveValue::customCSSText const):
(WebCore::CSSPrimitiveValue::equals const):
(WebCore::CSSPrimitiveValue::addDerivedHash const):
* Source/WebCore/css/CSSPrimitiveValue.h:
(WebCore::CSSPrimitiveValue::isFontIndependentLength):
(WebCore::CSSPrimitiveValue::isRootFontRelativeLength):
(WebCore::CSSPrimitiveValue::isFontRelativeLength):
(WebCore::CSSPrimitiveValue::isContainerPercentageLength):
(WebCore::CSSPrimitiveValue::isLength):
(WebCore::CSSPrimitiveValue::resolveAsNumber const):
(WebCore::CSSPrimitiveValue::resolveAsNumberNoConversionDataRequired const):
(WebCore::CSSPrimitiveValue::resolveAsNumberDeprecated const):
(WebCore::CSSPrimitiveValue::computeAngle):
(WebCore::CSSPrimitiveValue::computeTime):
(WebCore::CSSPrimitiveValue::computeFrequency):
(WebCore::CSSPrimitiveValue::computeResolution):
(WebCore::CSSPrimitiveValue::resolveAsLengthNoConversionDataRequired const):
(WebCore::CSSPrimitiveValue::resolveAsLengthDeprecated const):
* Source/WebCore/css/CSSPropertyInitialValues.cpp:
(WebCore::initialValueForLonghand):
(WebCore::isInitialValueForLonghand):
(WebCore::initialValueTextForLonghand):
* Source/WebCore/css/CSSUnits.cpp:
(WebCore::unitCategory):
(WebCore::canonicalUnitTypeForCategory):
(WebCore::operator&lt;&lt;):
(WebCore::conversionToCanonicalUnitsScaleFactor):
(WebCore::conversionToCanonicalUnitRequiresConversionData):
* Source/WebCore/css/CSSUnits.h:
* Source/WebCore/css/CSSValuePool.cpp:
(WebCore::StaticCSSValuePool::StaticCSSValuePool):
* Source/WebCore/css/DeprecatedCSSOMPrimitiveValue.cpp:
(WebCore::DeprecatedCSSOMPrimitiveValue::primitiveType const):
(WebCore::DeprecatedCSSOMPrimitiveValue::getFloatValue const):
* Source/WebCore/css/calc/CSSCalcTree+NumericIdentity.h:
(WebCore::CSSCalc::toNumericIdentity):
* Source/WebCore/css/calc/CSSCalcTree+Parser.cpp:
(WebCore::CSSCalc::parseCalcDimension):
* Source/WebCore/css/calc/CSSCalcTree+Serialization.cpp:
(WebCore::CSSCalc::sortPriority):
* Source/WebCore/css/calc/CSSCalcTree+Simplification.cpp:
(WebCore::CSSCalc::canonicalize):
* Source/WebCore/css/calc/CSSCalcTree.cpp:
(WebCore::CSSCalc::makeNumeric):
* Source/WebCore/css/calc/CSSCalcTree.h:
(WebCore::CSSCalc::toCSSUnit):
* Source/WebCore/css/calc/CSSCalcType.cpp:
(WebCore::CSSCalc::Type::determineType):
* Source/WebCore/css/calc/CSSCalcValue.cpp:
(WebCore::CSSCalc::Value::primitiveType const):
* Source/WebCore/css/parser/CSSParserFastPaths.cpp:
(WebCore::parseSimpleLength):
(WebCore::parseSimpleLengthValue):
(WebCore::parseColorIntOrPercentage):
(WebCore::parseNumericColor):
(WebCore::parseTransformTranslateArguments):
(WebCore::parseTransformNumberArguments):
(WebCore::parseOpacity):
* Source/WebCore/css/parser/CSSParserToken.cpp:
(WebCore::cssPrimitiveValueUnitFromTrie):
(WebCore::CSSParserToken::CSSParserToken):
(WebCore::CSSParserToken::convertToPercentage):
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Background.cpp:
(WebCore::CSSPropertyParserHelpers::consumeWebkitBoxReflect):
* Source/WebCore/css/parser/CSSPropertyParserConsumer+CSSPrimitiveValueResolver.h:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Color.cpp:
(WebCore::CSSPropertyParserHelpers::consumeRelativeFunctionParameters):
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Grid.cpp:
(WebCore::CSSPropertyParserHelpers::consumeGridBreadth):
* Source/WebCore/css/parser/CSSPropertyParserConsumer+LengthDefinitions.h:
(WebCore::CSSPropertyParserHelpers::LengthValidator::validate):
* Source/WebCore/css/parser/CSSPropertyParserCustom.h:
(WebCore::CSS::PropertyParserCustom::consumeFlexShorthand):
* Source/WebCore/css/typedom/CSSNumericFactory.h:
* Source/WebCore/css/typedom/CSSNumericValue.cpp:
(WebCore::invert):
(WebCore::CSSNumericValue::multiplyInternal):
(WebCore::createCSSUnitValueFromAddend):
(WebCore::CSSNumericValue::toSum):
* Source/WebCore/css/typedom/CSSStyleValueFactory.cpp:
(WebCore::CSSStyleValueFactory::reifyValue):
* Source/WebCore/css/typedom/CSSUnitValue.cpp:
(WebCore::CSSUnitValue::parseUnit):
(WebCore::CSSUnitValue::unit const):
(WebCore::CSSUnitValue::create):
(WebCore::CSSUnitValue::toSumValue const):
(WebCore::isValueOutOfRangeForProperty):
(WebCore::calculationCategoryForProperty):
* Source/WebCore/css/typedom/color/CSSOMColorValue.cpp:
(WebCore::CSSOMColorValue::rectifyCSSColorPercent):
(WebCore::CSSOMColorValue::rectifyCSSColorAngle):
(WebCore::CSSOMColorValue::rectifyCSSColorNumber):
* Source/WebCore/css/typedom/color/CSSRGB.cpp:
(WebCore::CSSRGB::rectifyCSSColorRGBComp):
* Source/WebCore/css/typedom/transform/CSSPerspective.cpp:
(WebCore::CSSPerspective::toMatrix):
* Source/WebCore/css/typedom/transform/CSSRotate.cpp:
(WebCore::CSSRotate::create):
(WebCore::CSSRotate::toMatrix):
* Source/WebCore/css/typedom/transform/CSSScale.cpp:
(WebCore::CSSScale::create):
* Source/WebCore/css/typedom/transform/CSSSkew.cpp:
(WebCore::CSSSkew::toMatrix):
* Source/WebCore/css/typedom/transform/CSSSkewX.cpp:
(WebCore::CSSSkewX::toMatrix):
* Source/WebCore/css/typedom/transform/CSSSkewY.cpp:
(WebCore::CSSSkewY::toMatrix):
* Source/WebCore/css/typedom/transform/CSSTranslate.cpp:
(WebCore::CSSTranslate::create):
(WebCore::CSSTranslate::toMatrix):
* Source/WebCore/css/values/color/CSSRelativeColorResolver.h:
(WebCore::CSS::resolve):
(WebCore::CSS::resolveNoConversionDataRequired):
* Source/WebCore/css/values/primitives/CSSPrimitiveNumericUnits.h:
(WebCore::CSS::toCSSUnitType):
(WebCore::CSS::toIntegerUnit):
(WebCore::CSS::toNumberUnit):
(WebCore::CSS::toPercentageUnit):
(WebCore::CSS::toAngleUnit):
(WebCore::CSS::toLengthUnit):
(WebCore::CSS::toTimeUnit):
(WebCore::CSS::toFrequencyUnit):
(WebCore::CSS::toResolutionUnit):
(WebCore::CSS::toFlexUnit):
(WebCore::CSS::toAnglePercentageUnit):
(WebCore::CSS::toLengthPercentageUnit):
* Source/WebCore/dom/ImageOverlay.cpp:
(WebCore::ImageOverlay::fitElementToQuad):
(WebCore::ImageOverlay::updateWithTextRecognitionResult):
* Source/WebCore/dom/ViewTransition.cpp:
(WebCore::ViewTransition::copyElementBaseProperties):
* Source/WebCore/editing/ApplyStyleCommand.cpp:
(WebCore::ApplyStyleCommand::applyRelativeFontStyleChange):
* Source/WebCore/editing/FontAttributeChanges.cpp:
(WebCore::FontChanges::createStyleProperties const):
* Source/WebCore/editing/cocoa/NodeHTMLConverter.mm:
(stringFromCSSValue):
(floatValueFromPrimitiveValue):
* Source/WebCore/editing/mac/EditorMac.mm:
(WebCore::Editor::platformPasteFont):
* Source/WebCore/html/HTMLAttachmentElement.cpp:
(WebCore::HTMLAttachmentElement::insertedIntoAncestor):
* Source/WebCore/html/HTMLElement.cpp:
(WebCore::HTMLElement::applyBorderAttributeToStyle):
(WebCore::HTMLElement::addHTMLLengthToStyle):
* Source/WebCore/html/HTMLEmbedElement.cpp:
(WebCore::HTMLEmbedElement::collectPresentationalHintsForAttribute):
* Source/WebCore/html/HTMLHRElement.cpp:
(WebCore::HTMLHRElement::collectPresentationalHintsForAttribute):
* Source/WebCore/html/HTMLIFrameElement.cpp:
(WebCore::HTMLIFrameElement::collectPresentationalHintsForAttribute):
* Source/WebCore/html/HTMLMarqueeElement.cpp:
(WebCore::HTMLMarqueeElement::collectPresentationalHintsForAttribute):
* Source/WebCore/html/HTMLMeterElement.cpp:
(WebCore::HTMLMeterElement::didElementStateChange):
* Source/WebCore/html/HTMLTableElement.cpp:
(WebCore::HTMLTableElement::collectPresentationalHintsForAttribute):
(WebCore::HTMLTableElement::createSharedCellStyle const):
* Source/WebCore/html/ImageDocument.cpp:
(WebCore::ImageDocument::createDocumentStructure):
* Source/WebCore/html/ValidationMessage.cpp:
(WebCore::ValidationMessage::adjustBubblePosition):
* Source/WebCore/html/shadow/DateTimeEditElement.cpp:
(WebCore::DateTimeEditBuilder::visitLiteral):
* Source/WebCore/html/shadow/MediaControlTextTrackContainerElement.cpp:
(WebCore::MediaControlTextTrackContainerElement::updateTextStrokeStyle):
(WebCore::MediaControlTextTrackContainerElement::updateTextTrackStyle):
* Source/WebCore/html/shadow/ProgressShadowElement.cpp:
(WebCore::ProgressValueElement::setInlineSizePercentage):
* Source/WebCore/html/shadow/SpatialImageControls.cpp:
(WebCore::SpatialImageControls::ensureSpatialControls):
* Source/WebCore/html/shadow/TextControlInnerElements.cpp:
(WebCore::TextControlInnerElement::resolveCustomStyle):
* Source/WebCore/html/shadow/TextPlaceholderElement.cpp:
* Source/WebCore/html/track/VTTCue.cpp:
(WebCore::VTTCueBox::applyCSSPropertiesWithRegion):
(WebCore::VTTCueBox::applyCSSProperties):
* Source/WebCore/html/track/VTTRegion.cpp:
(WebCore::VTTRegion::displayLastTextTrackCueBox):
(WebCore::VTTRegion::willRemoveTextTrackCueBox):
(WebCore::VTTRegion::prepareRegionDisplayTree):
* Source/WebCore/page/DragController.cpp:
(WebCore::DragController::insertDroppedImagePlaceholdersAtCaret):
* Source/WebCore/rendering/RenderLayer.cpp:
(WebCore::RenderLayer::resize):
* Source/WebCore/rendering/cocoa/RenderThemeCocoa.mm:
(WebCore::shapeForSearchField):
(WebCore::RenderThemeCocoa::adjustInnerSpinButtonStyleForVectorBasedControls const):
(WebCore::applyEmPadding):
(WebCore::paddingBoxForNumberField):
(WebCore::applyCommonButtonPaddingToStyleForVectorBasedControls):
(WebCore::RenderThemeCocoa::adjustButtonStyleForVectorBasedControls const):
(WebCore::RenderThemeCocoa::paintMenuListButtonDecorationsForVectorBasedControls):
(WebCore::RenderThemeCocoa::adjustSearchFieldCancelButtonStyleForVectorBasedControls const):
(WebCore::RenderThemeCocoa::adjustSearchFieldDecorationPartStyleForVectorBasedControls const):
* Source/WebCore/rendering/ios/RenderThemeIOS.mm:
(WebCore::RenderThemeIOS::popupInternalPaddingBox const):
(WebCore::applyCommonButtonPaddingToStyle):
(WebCore::RenderThemeIOS::paintMenuListButtonDecorations):
(WebCore::RenderThemeIOS::adjustButtonStyle const):
(WebCore::RenderThemeIOS::adjustSearchFieldDecorationPartStyle const):
* Source/WebCore/rendering/style/StyleImageSet.cpp:
(WebCore::StyleImageSet::computedStyleValue const):
* Source/WebCore/style/StyleExtractor.cpp:
(WebCore::Style::Extractor::getFontSizeCSSValuePreferringKeyword const):
* Source/WebCore/style/StyleExtractorConverter.h:
(WebCore::Style::ExtractorConverter::convertNumberAsPixels):
* Source/WebCore/style/values/box/StyleMargin.cpp:
(WebCore::Style::CSSValueConversion&lt;MarginEdge&gt;::operator):
* Source/WebCore/style/values/non-standard/StyleWebKitTextStrokeWidth.cpp:
(WebCore::Style::CSSValueConversion&lt;WebkitTextStrokeWidth&gt;::operator):
* Source/WebCore/style/values/primitives/StyleLengthWrapper+CSSValueConversion.h:
(WebCore::Style::CSSValueConversion&lt;T&gt;::operator()):
* Source/WebCore/style/values/primitives/StyleLengthWrapper+DeprecatedCSSValueConversion.h:
(WebCore::Style::DeprecatedCSSValueConversion&lt;T&gt;::operator()):
* Source/WebCore/style/values/text/StyleLetterSpacing.cpp:
(WebCore::Style::CSSValueConversion&lt;LetterSpacing&gt;::operator):
* Source/WebCore/style/values/text/StyleWordSpacing.cpp:
(WebCore::Style::CSSValueConversion&lt;WordSpacing&gt;::operator):
* Source/WebKit/WebProcess/Plugins/PDF/PDFPluginAnnotation.mm:
(WebKit::PDFPluginAnnotation::updateGeometry):
* Source/WebKit/WebProcess/Plugins/PDF/PDFPluginChoiceAnnotation.mm:
(WebKit::PDFPluginChoiceAnnotation::updateGeometry):
* Source/WebKit/WebProcess/Plugins/PDF/PDFPluginTextAnnotation.mm:
(WebKit::PDFPluginTextAnnotation::updateGeometry):
* Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm:
(WebKit::UnifiedPDFPlugin::sizeToFitContentsIfNeeded):
</pre><!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/0c90030cc272e6a9aa802b43c3b17b39dfc0f635

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/127897 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/171 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/38722 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/135273 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/79444 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/5aebc8f8-134b-4cef-987e-f07241c7f150) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/129769 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/64 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/156/builds/49 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/97359 "Passed tests") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/65444 "") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/8b6495c3-a0df-4d8c-9ec6-2f512e9363fa) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/130845 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/162/builds/20 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/114572 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/77925 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/0dbc4151-e635-45e2-a795-d83377e0d4ad) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/154/builds/33 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/32677 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/78581 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/108385 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/33147 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/137757 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/57 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/161/builds/42 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/105892 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/73 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/110918 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/105625 "Passed tests") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/29 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/29495 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/52199 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/96 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/61545 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/157/builds/35 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/163/builds/110 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/77 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/9d2af7b5c79b043c997724d6800f3afabacc1aff. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/32490

zakariaridouh committed:
Rename enums in CSSUnitType to CamelCase and remove CSS_ prefix
<https://bugs.webkit.org/show_bug.cgi?id=229622>
<rdar://82734077>

Reviewed by NOBODY (OOPS!).

Expand the names of most enums, and update their uses across the
codebase.
* CSS_S is renamed to Second etc...
* Cm => Centimeter and In => Inch etc...

* Source/WebCore/animation/KeyframeEffect.cpp:
(WebCore::isTimelineRangeOffsetValid):
(WebCore::computeMissingKeyframeOffsets):
(WebCore::specifiedOffsetForParsedKeyframe):
* Source/WebCore/animation/WebAnimationTime.cpp:
(WebCore::WebAnimationTime::WebAnimationTime):
* Source/WebCore/css/CSSImageSetOptionValue.cpp:
(WebCore::CSSImageSetOptionValue::create):
* Source/WebCore/css/CSSPrimitiveValue.cpp:
(WebCore::isValidCSSUnitTypeForDoubleConversion):
(WebCore::isStringType):
(WebCore::CSSPrimitiveValue::primitiveType const):
(WebCore::CSSPrimitiveValue::CSSPrimitiveValue):
(WebCore::CSSPrimitiveValue::~CSSPrimitiveValue):
(WebCore::CSSPrimitiveValue::create):
(WebCore::CSSPrimitiveValue::createFontFamily):
(WebCore::CSSPrimitiveValue::createInteger):
(WebCore::CSSPrimitiveValue::conversionToCanonicalUnitsScaleFactor):
(WebCore::CSSPrimitiveValue::doubleValueDividingBy100IfPercentage const):
(WebCore::CSSPrimitiveValue::doubleValueDividingBy100IfPercentageDeprecated const):
(WebCore::CSSPrimitiveValue::doubleValueInternal const):
(WebCore::CSSPrimitiveValue::doubleValueInternalDeprecated const):
(WebCore::CSSPrimitiveValue::stringValue const):
(WebCore::CSSPrimitiveValue::unitTypeString):
(WebCore::CSSPrimitiveValue::serializeInternal const):
(WebCore::CSSPrimitiveValue::customCSSText const):
(WebCore::CSSPrimitiveValue::equals const):
(WebCore::CSSPrimitiveValue::addDerivedHash const):
* Source/WebCore/css/CSSPrimitiveValue.h:
(WebCore::CSSPrimitiveValue::isFontIndependentLength):
(WebCore::CSSPrimitiveValue::isRootFontRelativeLength):
(WebCore::CSSPrimitiveValue::isFontRelativeLength):
(WebCore::CSSPrimitiveValue::isContainerPercentageLength):
(WebCore::CSSPrimitiveValue::isLength):
(WebCore::CSSPrimitiveValue::resolveAsNumber const):
(WebCore::CSSPrimitiveValue::resolveAsNumberNoConversionDataRequired const):
(WebCore::CSSPrimitiveValue::resolveAsNumberDeprecated const):
(WebCore::CSSPrimitiveValue::computeAngle):
(WebCore::CSSPrimitiveValue::computeTime):
(WebCore::CSSPrimitiveValue::computeFrequency):
(WebCore::CSSPrimitiveValue::computeResolution):
(WebCore::CSSPrimitiveValue::resolveAsLengthNoConversionDataRequired const):
(WebCore::CSSPrimitiveValue::resolveAsLengthDeprecated const):
* Source/WebCore/css/CSSPropertyInitialValues.cpp:
(WebCore::initialValueForLonghand):
(WebCore::isInitialValueForLonghand):
(WebCore::initialValueTextForLonghand):
* Source/WebCore/css/CSSUnits.cpp:
(WebCore::unitCategory):
(WebCore::canonicalUnitTypeForCategory):
(WebCore::operator<<):
(WebCore::conversionToCanonicalUnitsScaleFactor):
(WebCore::conversionToCanonicalUnitRequiresConversionData):
* Source/WebCore/css/CSSUnits.h:
* Source/WebCore/css/CSSValuePool.cpp:
(WebCore::StaticCSSValuePool::StaticCSSValuePool):
* Source/WebCore/css/DeprecatedCSSOMPrimitiveValue.cpp:
(WebCore::DeprecatedCSSOMPrimitiveValue::primitiveType const):
(WebCore::DeprecatedCSSOMPrimitiveValue::getFloatValue const):
* Source/WebCore/css/calc/CSSCalcTree+NumericIdentity.h:
(WebCore::CSSCalc::toNumericIdentity):
* Source/WebCore/css/calc/CSSCalcTree+Parser.cpp:
(WebCore::CSSCalc::parseCalcDimension):
* Source/WebCore/css/calc/CSSCalcTree+Serialization.cpp:
(WebCore::CSSCalc::sortPriority):
* Source/WebCore/css/calc/CSSCalcTree+Simplification.cpp:
(WebCore::CSSCalc::canonicalize):
* Source/WebCore/css/calc/CSSCalcTree.cpp:
(WebCore::CSSCalc::makeNumeric):
* Source/WebCore/css/calc/CSSCalcTree.h:
(WebCore::CSSCalc::toCSSUnit):
* Source/WebCore/css/calc/CSSCalcType.cpp:
(WebCore::CSSCalc::Type::determineType):
* Source/WebCore/css/calc/CSSCalcValue.cpp:
(WebCore::CSSCalc::Value::primitiveType const):
* Source/WebCore/css/parser/CSSParserFastPaths.cpp:
(WebCore::parseSimpleLength):
(WebCore::parseSimpleLengthValue):
(WebCore::parseColorIntOrPercentage):
(WebCore::parseNumericColor):
(WebCore::parseTransformTranslateArguments):
(WebCore::parseTransformNumberArguments):
(WebCore::parseOpacity):
* Source/WebCore/css/parser/CSSParserToken.cpp:
(WebCore::cssPrimitiveValueUnitFromTrie):
(WebCore::CSSParserToken::CSSParserToken):
(WebCore::CSSParserToken::convertToPercentage):
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Background.cpp:
(WebCore::CSSPropertyParserHelpers::consumeWebkitBoxReflect):
* Source/WebCore/css/parser/CSSPropertyParserConsumer+CSSPrimitiveValueResolver.h:
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Color.cpp:
(WebCore::CSSPropertyParserHelpers::consumeRelativeFunctionParameters):
* Source/WebCore/css/parser/CSSPropertyParserConsumer+Grid.cpp:
(WebCore::CSSPropertyParserHelpers::consumeGridBreadth):
* Source/WebCore/css/parser/CSSPropertyParserConsumer+LengthDefinitions.h:
(WebCore::CSSPropertyParserHelpers::LengthValidator::validate):
* Source/WebCore/css/parser/CSSPropertyParserCustom.h:
(WebCore::CSS::PropertyParserCustom::consumeFlexShorthand):
* Source/WebCore/css/typedom/CSSNumericFactory.h:
* Source/WebCore/css/typedom/CSSNumericValue.cpp:
(WebCore::invert):
(WebCore::CSSNumericValue::multiplyInternal):
(WebCore::createCSSUnitValueFromAddend):
(WebCore::CSSNumericValue::toSum):
* Source/WebCore/css/typedom/CSSStyleValueFactory.cpp:
(WebCore::CSSStyleValueFactory::reifyValue):
* Source/WebCore/css/typedom/CSSUnitValue.cpp:
(WebCore::CSSUnitValue::parseUnit):
(WebCore::CSSUnitValue::unit const):
(WebCore::CSSUnitValue::create):
(WebCore::CSSUnitValue::toSumValue const):
(WebCore::isValueOutOfRangeForProperty):
(WebCore::calculationCategoryForProperty):
* Source/WebCore/css/typedom/color/CSSOMColorValue.cpp:
(WebCore::CSSOMColorValue::rectifyCSSColorPercent):
(WebCore::CSSOMColorValue::rectifyCSSColorAngle):
(WebCore::CSSOMColorValue::rectifyCSSColorNumber):
* Source/WebCore/css/typedom/color/CSSRGB.cpp:
(WebCore::CSSRGB::rectifyCSSColorRGBComp):
* Source/WebCore/css/typedom/transform/CSSPerspective.cpp:
(WebCore::CSSPerspective::toMatrix):
* Source/WebCore/css/typedom/transform/CSSRotate.cpp:
(WebCore::CSSRotate::create):
(WebCore::CSSRotate::toMatrix):
* Source/WebCore/css/typedom/transform/CSSScale.cpp:
(WebCore::CSSScale::create):
* Source/WebCore/css/typedom/transform/CSSSkew.cpp:
(WebCore::CSSSkew::toMatrix):
* Source/WebCore/css/typedom/transform/CSSSkewX.cpp:
(WebCore::CSSSkewX::toMatrix):
* Source/WebCore/css/typedom/transform/CSSSkewY.cpp:
(WebCore::CSSSkewY::toMatrix):
* Source/WebCore/css/typedom/transform/CSSTranslate.cpp:
(WebCore::CSSTranslate::create):
(WebCore::CSSTranslate::toMatrix):
* Source/WebCore/css/values/color/CSSRelativeColorResolver.h:
(WebCore::CSS::resolve):
(WebCore::CSS::resolveNoConversionDataRequired):
* Source/WebCore/css/values/primitives/CSSPrimitiveNumericUnits.h:
(WebCore::CSS::toCSSUnitType):
(WebCore::CSS::toIntegerUnit):
(WebCore::CSS::toNumberUnit):
(WebCore::CSS::toPercentageUnit):
(WebCore::CSS::toAngleUnit):
(WebCore::CSS::toLengthUnit):
(WebCore::CSS::toTimeUnit):
(WebCore::CSS::toFrequencyUnit):
(WebCore::CSS::toResolutionUnit):
(WebCore::CSS::toFlexUnit):
(WebCore::CSS::toAnglePercentageUnit):
(WebCore::CSS::toLengthPercentageUnit):
* Source/WebCore/dom/ImageOverlay.cpp:
(WebCore::ImageOverlay::fitElementToQuad):
(WebCore::ImageOverlay::updateWithTextRecognitionResult):
* Source/WebCore/dom/ViewTransition.cpp:
(WebCore::ViewTransition::copyElementBaseProperties):
* Source/WebCore/editing/ApplyStyleCommand.cpp:
(WebCore::ApplyStyleCommand::applyRelativeFontStyleChange):
* Source/WebCore/editing/FontAttributeChanges.cpp:
(WebCore::FontChanges::createStyleProperties const):
* Source/WebCore/editing/cocoa/NodeHTMLConverter.mm:
(stringFromCSSValue):
(floatValueFromPrimitiveValue):
* Source/WebCore/editing/mac/EditorMac.mm:
(WebCore::Editor::platformPasteFont):
* Source/WebCore/html/HTMLAttachmentElement.cpp:
(WebCore::HTMLAttachmentElement::insertedIntoAncestor):
* Source/WebCore/html/HTMLElement.cpp:
(WebCore::HTMLElement::applyBorderAttributeToStyle):
(WebCore::HTMLElement::addHTMLLengthToStyle):
* Source/WebCore/html/HTMLEmbedElement.cpp:
(WebCore::HTMLEmbedElement::collectPresentationalHintsForAttribute):
* Source/WebCore/html/HTMLHRElement.cpp:
(WebCore::HTMLHRElement::collectPresentationalHintsForAttribute):
* Source/WebCore/html/HTMLIFrameElement.cpp:
(WebCore::HTMLIFrameElement::collectPresentationalHintsForAttribute):
* Source/WebCore/html/HTMLMarqueeElement.cpp:
(WebCore::HTMLMarqueeElement::collectPresentationalHintsForAttribute):
* Source/WebCore/html/HTMLMeterElement.cpp:
(WebCore::HTMLMeterElement::didElementStateChange):
* Source/WebCore/html/HTMLTableElement.cpp:
(WebCore::HTMLTableElement::collectPresentationalHintsForAttribute):
(WebCore::HTMLTableElement::createSharedCellStyle const):
* Source/WebCore/html/ImageDocument.cpp:
(WebCore::ImageDocument::createDocumentStructure):
* Source/WebCore/html/ValidationMessage.cpp:
(WebCore::ValidationMessage::adjustBubblePosition):
* Source/WebCore/html/shadow/DateTimeEditElement.cpp:
(WebCore::DateTimeEditBuilder::visitLiteral):
* Source/WebCore/html/shadow/MediaControlTextTrackContainerElement.cpp:
(WebCore::MediaControlTextTrackContainerElement::updateTextStrokeStyle):
(WebCore::MediaControlTextTrackContainerElement::updateTextTrackStyle):
* Source/WebCore/html/shadow/ProgressShadowElement.cpp:
(WebCore::ProgressValueElement::setInlineSizePercentage):
* Source/WebCore/html/shadow/SpatialImageControls.cpp:
(WebCore::SpatialImageControls::ensureSpatialControls):
* Source/WebCore/html/shadow/TextControlInnerElements.cpp:
(WebCore::TextControlInnerElement::resolveCustomStyle):
* Source/WebCore/html/shadow/TextPlaceholderElement.cpp:
* Source/WebCore/html/track/VTTCue.cpp:
(WebCore::VTTCueBox::applyCSSPropertiesWithRegion):
(WebCore::VTTCueBox::applyCSSProperties):
* Source/WebCore/html/track/VTTRegion.cpp:
(WebCore::VTTRegion::displayLastTextTrackCueBox):
(WebCore::VTTRegion::willRemoveTextTrackCueBox):
(WebCore::VTTRegion::prepareRegionDisplayTree):
* Source/WebCore/page/DragController.cpp:
(WebCore::DragController::insertDroppedImagePlaceholdersAtCaret):
* Source/WebCore/rendering/RenderLayer.cpp:
(WebCore::RenderLayer::resize):
* Source/WebCore/rendering/cocoa/RenderThemeCocoa.mm:
(WebCore::shapeForSearchField):
(WebCore::RenderThemeCocoa::adjustInnerSpinButtonStyleForVectorBasedControls const):
(WebCore::applyEmPadding):
(WebCore::paddingBoxForNumberField):
(WebCore::applyCommonButtonPaddingToStyleForVectorBasedControls):
(WebCore::RenderThemeCocoa::adjustButtonStyleForVectorBasedControls const):
(WebCore::RenderThemeCocoa::paintMenuListButtonDecorationsForVectorBasedControls):
(WebCore::RenderThemeCocoa::adjustSearchFieldCancelButtonStyleForVectorBasedControls const):
(WebCore::RenderThemeCocoa::adjustSearchFieldDecorationPartStyleForVectorBasedControls const):
* Source/WebCore/rendering/ios/RenderThemeIOS.mm:
(WebCore::RenderThemeIOS::popupInternalPaddingBox const):
(WebCore::applyCommonButtonPaddingToStyle):
(WebCore::RenderThemeIOS::paintMenuListButtonDecorations):
(WebCore::RenderThemeIOS::adjustButtonStyle const):
(WebCore::RenderThemeIOS::adjustSearchFieldDecorationPartStyle const):
* Source/WebCore/rendering/style/StyleImageSet.cpp:
(WebCore::StyleImageSet::computedStyleValue const):
* Source/WebCore/style/StyleExtractor.cpp:
(WebCore::Style::Extractor::getFontSizeCSSValuePreferringKeyword const):
* Source/WebCore/style/StyleExtractorConverter.h:
(WebCore::Style::ExtractorConverter::convertNumberAsPixels):
* Source/WebCore/style/values/box/StyleMargin.cpp:
(WebCore::Style::CSSValueConversion<MarginEdge>::operator):
* Source/WebCore/style/values/non-standard/StyleWebKitTextStrokeWidth.cpp:
(WebCore::Style::CSSValueConversion<WebkitTextStrokeWidth>::operator):
* Source/WebCore/style/values/primitives/StyleLengthWrapper+CSSValueConversion.h:
(WebCore::Style::CSSValueConversion<T>::operator()):
* Source/WebCore/style/values/primitives/StyleLengthWrapper+DeprecatedCSSValueConversion.h:
(WebCore::Style::DeprecatedCSSValueConversion<T>::operator()):
* Source/WebCore/style/values/text/StyleLetterSpacing.cpp:
(WebCore::Style::CSSValueConversion<LetterSpacing>::operator):
* Source/WebCore/style/values/text/StyleWordSpacing.cpp:
(WebCore::Style::CSSValueConversion<WordSpacing>::operator):
* Source/WebKit/WebProcess/Plugins/PDF/PDFPluginAnnotation.mm:
(WebKit::PDFPluginAnnotation::updateGeometry):
* Source/WebKit/WebProcess/Plugins/PDF/PDFPluginChoiceAnnotation.mm:
(WebKit::PDFPluginChoiceAnnotation::updateGeometry):
* Source/WebKit/WebProcess/Plugins/PDF/PDFPluginTextAnnotation.mm:
(WebKit::PDFPluginTextAnnotation::updateGeometry):
* Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm:
(WebKit::UnifiedPDFPlugin::sizeToFitContentsIfNeeded):

nt1m commented on Source/WebCore/css/parser/CSSParserToken.cpp:
```diff
@@ -47,79 +47,79 @@ CSSUnitType cssPrimitiveValueUnitFromTrie(std::span<const CharacterType> data)
+                return CSSUnitType::Ch;
             case 'm':
-                return CSSUnitType::CSS_CM;
+                return CSSUnitType::Centimeter;
```
I prefer whatever web developers use, which are abbreviations in this case
